bombbot.py

```diff
@@ -31,7 +31,7 @@
     'TARGET_BOT':             "You thought you could trick me into bombing myself?!",
     'TARGET_SELF':            "%s pls. Bomb a friend if you have to!",
     'TARGET_IMAGINARY':       "You can't bomb imaginary people!",
-    'TARGET_DISABLED':        "I'm not allowed to bomb %s, sorry.",
```

Comment:
`%s` was removed here, but the value interpolation was not removed from places where the string is used.